### PR TITLE
[TF] Sanity test for Mask R-CNN

### PR DESCRIPTION
**Pull request by andrey-churkin**

### CODE REVIEW

File: beta/examples/tensorflow/classification/main.py [hunk 1]
```diff
@@ -70,8 +70,7 @@ def get_config_from_argv(argv, parser):
     return config
 
 
-def get_dataset_builders(config, strategy, one_hot=True):
-    num_devices = strategy.num_replicas_in_sync if strategy else 1
+def get_dataset_builders(config, num_devices, one_hot=True):
```
Comment (alexsu52) [inline beta/examples/tensorflow/classification/main.py:73]: For what reason are you changing the signature of the get_dataset_builders function?

File: beta/examples/tensorflow/segmentation/train.py [hunk 2]
```diff
@@ -46,6 +47,11 @@ def get_argument_parser():
                                         print_freq=False,
                                         dataset_type=False,
                                         cpu_only=False)
+    parser.add_argument('--mode', '-m',
```
Comment (alexsu52) [inline beta/examples/tensorflow/segmentation/train.py:50]: Seems, this is a workaround for the sanity test.

**Review by alexsu52 — CHANGES_REQUESTED** [review]

File: beta/examples/tensorflow/segmentation/train.py (hunk 2 above)
Comment (andrey-churkin) [reply beta/examples/tensorflow/segmentation/train.py]: Yes, I fixed it.

File: beta/examples/tensorflow/classification/main.py (hunk 1 above)
Comment (andrey-churkin) [reply beta/examples/tensorflow/classification/main.py:73]: These minor changes are needed to make dependencies more explicit.

**Comment by andrey-churkin** [comment]
Jenkins please retry a build

**Review by alexsu52 — APPROVED** [review]

**Pull request merged**